Utils/Dataflow/030_PDFAnalyzer/manager.py

```diff
@@ -30,7 +30,8 @@ def load_config(default_cfg):
             loaded_cfg = json.load(f)
     except Exception as e:
         sys.stderr.write("Exception while loading config: %s\n"%e)
-        sys.stderr.write("No config file loaded, using default values\n")
+        sys.stderr.write("No config file loaded, using default values\

```

Comment:
`^M` again, and it seems to be added into the log message.
Use 
```
sys.stderr.write("No config file loaded, using default values"
                 "\n")
```
instead, or else you will have the two-string message: with text and with spaces.
The newline within quotes is just the same as `\n`.